src/hooks/useUnreadMessages.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {
+  useState,
+  createContext,
+  useContext,
+  useEffect,
+  useCallback,
+} from 'react';
+import { uniq } from 'lodash';
+import { useAsyncStorage } from './useAsyncStorage';
+import { useNotificationManager } from './useNotificationManager';
+
+type UnreadMessageContextProps = {
+  unreadMessagesUserIds?: string[];
+  markMessageRead?: (userId: string) => void;
+};
+
+const UnreadMessagesContext = createContext<UnreadMessageContextProps>({});
+
+export const UnreadMessagesContextProvider = ({
```

Comment:
This hook is crashing for me.  I'm looking into it...

<img width="744" alt="Screenshot 2023-08-30 at 2 21 01 PM" src="https://github.com/lifeomic/react-native-sdk/assets/220893/d55c0683-adfe-42eb-bdad-135294b115d0">

Reply:
Interesting, I didn't hit this. I wonder if something went wrong with the rebase.

Reply:
That is likely.  I put up [this PR](https://github.com/lifeomic/react-native-sdk/pull/350) that fixes it

Reply:
Merged! Thank you.